src/pyfmi/fmi_algorithm_drivers.py

```diff
@@ -1046,7 +1066,9 @@ def solve(self):
 
         #For result writing
         start_time_point = timer()
+        # Always log the start even if we are downsampling
         result_handler.integration_point()
+        self.downsampling_counter += 1
```

Comment:
I always also thinking of updating this by calling the new function, with some kind of boolean argument "force_integration_point", then all the calls to integration_point will be via the new function.

Reply:
I think this should not be required; one can always use the ordinary `self.result_handler.integration_point()` for that